### ISSUE TITLE
Link to multiple purchasing sources for all books.

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#2]
All books listed in the curriculum should include as many sourcing options as possible for purchasing the books. Where available, minimum should include where students can buy on Amazon, MAPS store, and direct from the publisher.

**Comment by CheekeyMonkey** [comment]
Could I suggest linking the MAPS bookstore in preference to Amazon? As buying their books will support a NFP.

**Comment by unshakenme** [comment]
Yes but not all books are available through the MAPS bookstore. The idea was to give people options rather than being gatekeeper.

We could suggest buying books from MAPS for that reason, but still provide alternatives.

**Comment by CheekeyMonkey** [comment]
Yes it's just the way the links were set in the file. I'm not sure if it
was purposefully alphabetical i.e. Amazon first, then Maps, perhaps
changing them over or putting NFP / Charity next to MAPS?

**Comment by a-n-g-i-e-r-i** [comment]
Highly recommend linking to https://www.bookfinder.com/ for each book. It is a price based book comparison tool that aggregates many websites. Could also include https://www.indiebound.org/ links when available.
